assembl/static2/js/app/components/debate/thread/video.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { I18n } from 'react-redux-i18n';
+import { Grid, Row, Col } from 'react-bootstrap';
+
+class Video extends React.Component {
```

Comment:
Didn't this component exist before? Could you not re-use the phase1 component?

Reply:
https://github.com/assembl/assembl/pull/161#discussion_r138285088